failure real (but just need to update the test):
```
======================================================================
FAIL: test_get_all_doc_ids_for_domain_grouped_by_db (corehq.couchapps.tests.test_all_docs.AllDocsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/couchapps/tests/test_all_docs.py", line 35, in test_get_all_doc_ids_for_domain_grouped_by_db
    get_db('fixtures').uri: []}
AssertionError: {'http://127.0.0.1:5984/commcarehq__meta_test': [], 'http://127.0.0.1:5984/commc [truncated]... != {'http://127.0.0.1:5984/commcarehq__meta_test': [], 'http://127.0.0.1:5984/commc [truncated]...
- {'http://127.0.0.1:5984/commcarehq__domains_test': [],
-  'http://127.0.0.1:5984/commcarehq__fixtures_test': [],
? ^
+ {'http://127.0.0.1:5984/commcarehq__fixtures_test': [],
? ^
   'http://127.0.0.1:5984/commcarehq__meta_test': [],
   'http://127.0.0.1:5984/commcarehq__users_test': ['users_db_doc'],
   'http://127.0.0.1:5984/commcarehq_test': ['main_db_doc']}
----------------------------------------------------------------------
```

--- a/corehq/couchapps/tests/test_all_docs.py
+++ b/corehq/couchapps/tests/test_all_docs.py
@@ -32,7 +32,8 @@ class AllDocsTest(TestCase):
             {get_db(None).uri: ['main_db_doc'],
              get_db('users').uri: ['users_db_doc'],
              get_db('meta').uri: [],
-             get_db('fixtures').uri: []}
+             get_db('fixtures').uri: [],
+             get_db('domains').uri: []}
         )
 
     def test_get_doc_count_by_type(self):

